embedder-tests/savant/src/test/resources/logback-test.xml

```diff
@@ -26,9 +26,9 @@
   </appender>
   
   <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-    <file>${indy.home}/var/log/indy/indy.log</file>
+    <file>target/var/log/indy/indy.log</file>
```

Comment:
It is not related to new added code here. I'm not sure what happened why there always is a "indy.home_IS_UNDEFINED" directory after a ftest build in embedder-test/savant directory, seems some ftests not located to direct ${indy.home} directory. So I just simply change it to this.
BTW, seems all log files are not in this directory for ftests, which is really in failsafe-reports/***-out.txt